src/index.html

```diff
@@ -18,14 +18,14 @@ <h1>Leaderboard</h1>
       <article>
         <div id="scores-section-title" class="flex">
           <h3>Recent scores</h3>
-          <button>Refresh</button>
+          <button type="button" class="refresh">Refresh</button>
         </div>
```

Comment:
- [ ] Kindly note that for this project, you hardly committed, 🙂 Committing frequently gives you the advantage of troubleshooting commits that have broken something in the build process and being able to revert back without losing too much work. You also get the added advantage of seeing changes easier without wading through a file with 1000 line additions. Check this [link](https://ilikekillnerds.com/2014/11/commit-often-merge-often-push-often-and-relax/#:~:text=Committing%20frequently%20gives%20you%20the,file%20with%201000%20line%20additions.) for a short interesting article about git commits, merges and pushes. it's sweet I tell you 😀.